WalletWasabi.Fluent/Views/Wallets/WalletView.axaml

```diff
@@ -69,84 +73,158 @@
+                <Setter Property="Grid.RowSpan" Value="{Binding RowSpan}" />
+                <Setter Property="IsVisible" Value="{Binding IsVisible}" />
+              </Style>
+            </ItemsControl.Styles>
+            <ItemsControl.ItemsPanel>
+              <ItemsPanelTemplate>
+                <c:BindableGrid ColumnDefinitionsSource="{Binding CurrentLayout.ColumnDefinitions}"
+                                RowDefinitionsSource="{Binding CurrentLayout.RowDefinitions}"
+                                ShowGridLines="False" />
+              </ItemsPanelTemplate>
+            </ItemsControl.ItemsPanel>
+            <ItemsControl.ItemTemplate>
+              <locator:ViewLocator />
+            </ItemsControl.ItemTemplate>
+          </ItemsControl>
+        </ScrollViewer>
+        <Border ClipToBounds="False"
+                CornerRadius="5"
+                Background="{DynamicResource TileRegionColor}"
+                Margin="6"
+                Padding="16"
+                BoxShadow="{DynamicResource InfoBoxShadow}"
+                IsVisible="{Binding IsEmptyWallet}">
+          <StackPanel>
+            <Label Content="Since this is a empty wallet, there isn't a lot going on here. Click below to receive Bitcoin."
+                   Classes="wrap center"
+                   MaxWidth="400"
+                   HorizontalAlignment="Center"
+                   VerticalAlignment="Center"/>
+              <Button Classes="function"
+                      Command="{Binding ReceiveCommand}"
+                      HorizontalAlignment="Center"
+                      VerticalAlignment="Center">
+              <StackPanel Orientation="Horizontal" Spacing="15">
+                <PathIcon Data="{StaticResource wallet_action_receive}" />
+                <TextBlock Text="Receive" />
+              </StackPanel>
+            </Button>
+          </StackPanel>
+        </Border>
+      </Panel>
+      <Panel>
+        <history:HistoryTable Name="HistoryTable"
+                              DataContext="{Binding History}"
+                              HorizontalAlignment="Stretch" VerticalAlignment="Stretch"
+                              IsVisible="{Binding !IsTransactionHistoryEmpty}"/>
+        <Border ClipToBounds="False"
+                CornerRadius="5"
+                Background="{DynamicResource TileRegionColor}"
+                Margin="6"
+                Padding="16"
+                BoxShadow="{DynamicResource InfoBoxShadow}"
+                DataContext="{Binding History}"
+                IsVisible="{Binding IsTransactionHistoryEmpty}">
+          <StackPanel HorizontalAlignment="Center" VerticalAlignment="Center">
+            <Image Width="234" Height="110" HorizontalAlignment="Center" VerticalAlignment="Center">
+              <Image.Source>
+                <DrawingImage Drawing="{StaticResource history_table_empty_wallet}"/>
+              </Image.Source>
+            </Image>
```

Comment:
Should be inside of a `Viewbox` to get it to work on the smallest size. Prob parent should be DockPanel then?

Reply:
Added Viewbox